linode_api4/linode_client.py

```diff
@@ -871,6 +871,205 @@ def users(self, *filters):
+        """
+        Returns a collection of Notification objects representing important, often time-sensitive items related to your Account.
+
+        API Documentation: https://www.linode.com/docs/api/account/#notifications-list
+
+        :returns: A list of Notifications on this account.
+        :rtype: List of Notification objects as MappedObjects
+        """
+
+        result = self.client.get(
+            "{}/notifications".format(Account.api_endpoint), model=self
+        )
+
+        return [MappedObject(**r) for r in result["data"]]
+
+    def linode_managed_enable(self):
+        """
+        Enables Linode Managed for the entire account and sends a welcome email to the account’s associated email address.
+
+        API Documentation: https://www.linode.com/docs/api/account/#linode-managed-enable
+        """
+
+        resp = self.client.post(
+            "{}/settings/managed-enable".format(Account.api_endpoint),
+            model=self,
+        )
+
+        if "errors" in resp:
+            raise UnexpectedResponseError(
+                "Unexpected response when enabling Linode Managed!",
+                json=resp,
+            )
+
+    def add_promo_code(self, promo_code):
+        """
+        Adds an expiring Promo Credit to your account.
+
+        API Documentation: https://www.linode.com/docs/api/account/#promo-credit-add
+
+        :param promo_code: The Promo Code.
+        :type promo_code: str
+        """
+
+        params = {
+            "promo_code": promo_code,
+        }
+
+        resp = self.client.post(
+            "{}/promo-codes".format(Account.api_endpoint),
+            model=self,
+            data=params,
+        )
+
+        if "errors" in resp:
+            raise UnexpectedResponseError(
+                "Unexpected response when adding Promo Code!",
+                json=resp,
+            )
+
+    def service_transfers(self):
+        """
+        Returns a collection of all created and accepted Service Transfers for this account, regardless of the user that created or accepted the transfer.
+
+        API Documentation: https://www.linode.com/docs/api/account/#service-transfers-list
+
+        :returns: A list of Service Transfers on this account.
+        :rtype: PaginatedList of ServiceTransfer
+        """
+
+        return self.client._get_and_filter(ServiceTransfer)
+
+    def service_transfer_create(self, entities):
+        """
+        Creates a transfer request for the specified services.
+
+        API Documentation: https://www.linode.com/docs/api/account/#service-transfer-create
+
+        :param entities: A collection of the services to include in this transfer request, separated by type.
+        :type entities: dict
+
+        Example usage::
+           entities = {
+                "linodes": [
+                    111,
+                    222
+                ]
+            }
+        """
+
+        if not entities["entities"] or "linodes" not in entities["entities"]:
```

Comment:
Assuming `entities` is a dict with keys for each entity type, this validation step might not function as intended. I think we can drop the `["entities"]` here since the `entities` key is specified when the params dict is created.

We might also want to drop the validation that `linodes` is specified since there may be support for other types of entity transfers in the future 👍 

Side note: To further enforce client-side validation, maybe we could check whether all of the entity type keys map to lists:
```python
bad_entries = [k for k, v in entities.items() if not isinstance(v, list)]
if len(bad_entries) > 0:
    raise ValueError(f"Got unexpected type for entity lists: {', '.join(bad_entries)}")
```